### PR TITLE
fixed 6.0 'wait until server ready' step in upload packs flow

### PR DESCRIPTION
**Pull request by yaakovi**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
wait until server 6.0 is ready instead of server master.

### CODE REVIEW

**Review by guyfreund — APPROVED** [review]
I don't think we will be able to merge this before the scheduled run, so I'll trigger it after merging

File: .circleci/config.yml [hunk 1]
```diff
@@ -1182,7 +1182,12 @@ jobs:
       - *attach_workspace
       - *add_ssh_keys
       - *prepare_environment
-      - *wait_until_server_ready
+      - run:
+          name: Wait until server ready
+          shell: /bin/bash
+          when: always
+          command: |
+            python3 ./Tests/scripts/wait_until_server_ready.py "Demisto 6.0"
```
Comment (guyfreund) [inline .circleci/config.yml:1190]:
```suggestion
            python3 ./Tests/scripts/wait_until_server_ready.py "$INSTNACE_ROLE"
```

Comment (guyfreund) [reply .circleci/config.yml:1190]: I think we should refactor the `Install Packs in Demisto Marketplace - Master` job too

**Review by guyfreund — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (ShahafBenYakir) [reply .circleci/config.yml:1190]: @yaakovi Is the AMI name of server master is `Demisto Marketplace`? If it is no need to sit `Install Packs in Demisto Marketplace - Master`

**Review by ShahafBenYakir — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (yaakovi) [reply .circleci/config.yml:1190]: as @ShahafBenYakir said, no need to modify the second job since the step is using the correct role

**Pull request merged**